.github/workflows/gcr-snapshot.yml

```diff
@@ -27,4 +27,4 @@ jobs:
       - name: Deploy to Cloud Run
         run: |
           gcloud components install beta --quiet
-          gcloud beta run deploy ${{ secrets.GCLOUD_SNAPSHOT_APP_NAME }} --quiet --image gcr.io/${{ secrets.GCLOUD_PROJECT }}/${{ secrets.GCLOUD_SNAPSHOT_APP_NAME }}:snapshot --project ${{ secrets.GCLOUD_PROJECT }} --region  us-central1 --platform managed --allow-unauthenticated --service-account=${{ secrets.GCLOUD_EMAIL }}
+          gcloud run deploy ${{ secrets.GCLOUD_SNAPSHOT_APP_NAME }} --quiet --image gcr.io/${{ secrets.GCLOUD_PROJECT }}/${{ secrets.GCLOUD_SNAPSHOT_APP_NAME }}:snapshot --project ${{ secrets.GCLOUD_PROJECT }} --region  us-central1 --update-env-vars=HOSTNAME="launch.micronaut.io",GITHUB_OAUTH_APP_CLIENT_ID=${{ secrets.GH_OAUTH_APP_CLIENT_ID }},GITHUB_OAUTH_APP_CLIENT_SECRET=${{ secrets.GH_OAUTH_APP_CLIENT_SECRET }} --platform managed --allow-unauthenticated --service-account=${{ secrets.GCLOUD_EMAIL }}
```

Comment:
@graemerocher I had to insert here injection of hostname of the launcher, because the RequestInfo contained localhost:8080. I can check whether it doesn't contain X- headers so it can be taken from there, but this way, it follows what is configured in StarterConfiguration:
```
String hostname = System.getenv(Environment.HOSTNAME);
```

Reply:
@jameskleeh could you check if the production instance has any other environment variables set, since we may need to ensure these are set by this command